server.js

```diff
@@ -59,48 +63,81 @@ function initializeApplication() {
         });
     });
 
-    app.use('/', routes);
-    app.use('/api', api);
-    app.use('/', admin);
-    app.use('/', organizations);
-    app.use('/users', users);
+    /*
+      In order to have the app.use() calls in server.js, I need to make a call
+      to schools.findSchools and loop through the results. Unfortunately by the
+      time that finishes, the server has already started, but without our
+      subdomains. So I'm using async to run these last two chunks of code in
+      series.
+    */
+    async.series([
+      function(callback) { // create school subdomains
+        schools.findSchools(function(err, schools) {
+          if (err) {
+            return callback(err);
+          }
+          for (var i = 0; i < schools.length; i++) {
+            var schoolID = schools[i].shortName;
+            var pageText = schools[i].pageText;
+            schoolRoutes.createSchool(pageText, function(err, route) {
+              if (err) {
+                next(err);
```

Comment:
This line must be `return callback(err)` instead of `next(err)` because:
1.- there is no `next` function at this point (you want to call the next step in async.series)
2.- if you omit the return, the next line `app.use(...)` will be called and I don't think that's what you want.